src/main/java/fi/aalto/cs/apluscourses/intellij/actions/SubmitExerciseAction.java

```diff
@@ -282,9 +300,11 @@ private void trySubmit(@NotNull Project project)
         submission.getPresentableExerciseName(),
         submission.getCurrentSubmissionNumber());
     addLocalHistoryTag(project, tag);
+    logger.info("Finished submitting exercise (took {}ms)", stopwatch.getTimeMs());
```

Comment:
The time measurement is pointless here as it also includes time spent in the dialog window (but again, you shouldn't measure time, see other comments)